src/SupervisionSuccess/constants.js

```diff
@@ -0,0 +1,2 @@
+export const CSV_FETCHING_ERROR = "Error occurred while fetching csv file";
```

Comment:
Is this going to be used in a future PR? It doesn't appear to be used now.

Reply:
No, but in last version of PR there is no this const.